staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/celcoststability_test.go

```diff
@@ -1,3 +1,6 @@
+// +build !arm
+// +build !386
```

Comment:
go fmt wants this done a little different with two modes, please gofmt

/lgtm-cancel

Reply:
Done!